### ISSUE TITLE
ENH: simple scripts to create yaml's from list of fastqs/bams

### ISSUE THREAD

**Issue by brentp** [bcbio/bcbio-nextgen#22]
In addition to the yaml's in example/, it'd be nice to have templates that would expand for my full list of samples, so that I could run, e.g.:

```
python scripts/create-yaml.py --template examples/variant2-base.yaml *.bam > my.yaml
```

and have it create a reasonable yaml with the current best-practices for variant calling. This could also be e.g.:

```
python scripts/create-yaml.py --template examples/variant2-ensembl.yaml *.bam > my.ensembl.yaml
```

I'm happy to try to implement this, just opening for discussion to decide best way to proceed. E.g. how much should we have a separate template for each analysis vs how much to include in the script itself. 

Also things like, if the args are a list of fastqs, can we automatically pair them and create a yaml that will align them with bwa and then run variant2?

Basically, it'd be nice if there were more guidance on how to get things running given a list of input files.
